module/zfs/spa_misc.c

```diff
@@ -2194,9 +2197,9 @@ spa_history_list_t *spa_import_progress_list = NULL;
 static int
 spa_import_progress_show_header(struct seq_file *f)
 {
-	seq_printf(f, "%-20s %-14s %-14s %-12s %s\n", "pool_guid",
+	seq_printf(f, "%-20s %-14s %-14s %-12s %s %s\n", "pool_guid",
```

Comment:
```suggestion
	seq_printf(f, "%-20s %-14s %-14s %-12s %-16s %s\n", "pool_guid",
```

Let's pad the pool name out to 16 characters so the notes column content is aligned with the "notes" header.  At least for pool names which aren't that long.

Reply:
fixed